### PR TITLE
3993 – Improvements for Check API seed script: Add an item with multiple medias

### PR DESCRIPTION
**Pull request by vasconsaurus**
## Description

We now add an item of each media type with multiple medias – which means it has many confirmed relationships. We do this so the center column on the item page has a good size list to check functionality against. 

As per Brian’s [request](https://github.com/meedan/check-api/pull/1722#issuecomment-1798729043)

References: 3993

## How has this been tested?

Ran the seeds script to create and add data.

### CODE REVIEW

**Comment by vasconsaurus** [comment]
@brianfleming can you take a look at this one and see if there is anything else you need?

File: db/seeds.rb [hunk 1]
```diff
@@ -112,8 +112,13 @@ def create_claim_description_for_imported_fact_check(user, project, team)
...
 
-def create_confirmed_relationship(parent, child)
-  Relationship.create!(source_id: parent.id, target_id: child.id, relationship_type: Relationship.confirmed_type)
+def create_confirmed_relationship(parent, children)
+  if children.class == Array
+    children.each { |child| Relationship.create!(source_id: parent.id, target_id: child.id, relationship_type: Relationship.confirmed_type)}
+  else
+    child = children
+    Relationship.create!(source_id: parent.id, target_id: child.id, relationship_type: Relationship.confirmed_type)
+  end
```
Comment (caiosba) [inline db/seeds.rb:121]: If I'm reading this correctly, this method is basically handling the case when there can be one or more children?
If so, I suggestion the following rewrite (just a suggestion, no need to change):

```
[children].flatten.each { |child| Relationship.create!(source_id: parent.id, target_id: child.id, relationship_type: Relationship.confirmed_type) }
```

**Review by caiosba — APPROVED** [review]
I added a suggestion but not a blocker :)

**Review by melsawy — APPROVED** [review]
I left one comment  for simplicity and it's up to you to keep current one or use my suggestion

File: db/seeds.rb (hunk 1 above)
Comment (vasconsaurus) [reply db/seeds.rb:121]: I wondered about this one, will update it ☺️

**Comment by brianfleming** [comment]
so...much...media!!
Thanks @vasconsaurus , love it

highfive

**Pull request merged**